.github/workflows/actions/publish-npm/action.yml

```diff
@@ -22,6 +22,9 @@ runs:
     - uses: actions/setup-node@v3
       with:
         node-version: 16.x
+    - name: Install latest npm
+      run: npm install -g npm@latest
```

Comment:
Node 16 includes npm 8.19.4, but package provenance requires npm 9.5. Alternatively we could install Node 18.